src/assets/styles/colours.css

```diff
@@ -233,6 +233,16 @@ html {
     --colour-service-accent-1: var(--colour-apmt-accent-1);
     --colour-service-accent-2: var(--colour-apmt-accent-2);
   }
+  &[data-tbpro-service='send'] {
+    --colour-service-soft: var(--colour-primary-soft);
+    --colour-service-primary: var(--colour-primary-default);
+    --colour-service-primary-hover: var(--colour-primary-hover);
+    --colour-service-primary-pressed: var(--colour-primary-pressed);
+    --colour-service-secondary: var(--colour-neutral-subtle);
+    --colour-service-accent-1: var(--colour-send-primary-accent-1);
+    --colour-service-accent-2: var(--colour-send-primary-accent-2);
```

Comment:
Accent colours are temp because I don't think the colour guide had them at the time of making this file:
https://github.com/thunderbird/services-ui/pull/13/files#diff-d8e2b40416514d67541fea7682c794279a12c194fa05b1eb6bf004520fc48497R224